### ISSUE TITLE
newer versions of node?

### ISSUE THREAD

**Issue by agreiner** [tugraz-isds/diamond#47]
Hi, I'm trying to stand up an instance of Diamond using node 21.5.0, since node 16 is end of life. Should I be able to get it to work without modifying the code in this repo? So far, I've been able to get it to show me the login page, but I've not been able to log in as the admin or to register a new account. Checking the js console shows errors with the calls to the api and no payload in either request.

**Comment by agreiner** [comment]
I am apparently able to log in as the admin user, but the response is "Your account is not enabled."

**Comment by blumma** [comment]
I have now tested the current main branch using node v16, v20 (current lts) and v21. They should all work fine. I used the deploy/local docker-compose.yml to run the database. I suggest that you use at least the long term support (lts) version of node (v20). With the next release, we will update this.

Your admin account has to be named "admin" for compatibility reasons. Otherwise Diamond checks the "study" property of the user and if it is not set, the response "Your account is not enabled." will be displayed.  (See line 3 of [cardsort-studies.component.html](https://github.com/tugraz-isds/diamond/blob/main/client/src/app/cardsort-components/cardsort-studies/cardsort-studies.component.html) and line 64/90 of [authentification.service.ts](https://github.com/tugraz-isds/diamond/blob/main/client/src/app/authentification.service.ts))
